### DISCUSSION TITLE
Epoch setting question

### DISCUSSION THREAD

**Discussion by vedicsharma** [Project-MONAI/MONAILabel#1717] [Q&A] [answered]
Hello, I had a quick question regarding Monai label. I notice that for a project I am collaborating on the main.py file for the active learning segmentation project had a setting of 10 epochs; however, 50 epochs ended up being used, as referenced in the logs and the paper itself. I was just wondering where this change can be made? Is there a GUI setting for this, or was this a parameter in another file, perhaps a training file? I am trying to understand the code of the project and what was done, and this information would be helpful. Thank you!

**Comment by diazandr3s** [comment, marked as answer]
Hi @vedicsharma,

Good question.

It should be possible to set the number of epochs via the Slicer module. Otherwise you could change it here in the parent **_basic_train_** class: https://github.com/Project-MONAI/MONAILabel/blob/main/monailabel/tasks/train/basic_train.py#L161

**_basic_train_** is the class inherited by the trainers file: https://github.com/Project-MONAI/MONAILabel/blob/main/sample-apps/radiology/lib/trainers/segmentation.py#L41

Hope this helps,
